openshift/templates/huginn-mysql.json

```diff
@@ -102,7 +102,7 @@
                     "from": {
                         "kind": "ImageStreamTag",
                         "namespace": "${NAMESPACE}",
-                        "name": "ruby:2.3"
+                        "name": "ruby:latest"
```

Comment:
Can we keep the explicit ruby version here? The latest ruby is `2.7`, but we only test against `2.6` at the moment.

Reply:
Yes, we can.  4.5 is the latest in the OpenShift 4.2 registry for the S2I builder's Ruby image, so Updated accordingly.

Reply:
Thanks! I am assuming that this `ruby:4.5` image is using a ruby version that is supported by Huginn. Can you provide a link to the openshift images for future references? I tried googling it without success 😞

Reply:
@dsander Sure!
For ruby-2.5 specifically: https://github.com/sclorg/s2i-ruby-container/blob/master/2.5/README.md
https://github.com/sclorg/s2i-ruby-container for the main repo and other versions floating around

Reply:
might also be helpful to see: https://www.softwarecollections.org/en/scls/rhscl/rh-ruby25/